### PR TITLE
Use superscripts in math mode for km/m3

### PR DESCRIPTION
**Pull request by PaulWessel**
As it says.

### CODE REVIEW

File: doc/rst/source/supplements/potential/gravprisms.rst [hunk 1]
```diff
@@ -76,7 +76,7 @@ Required Arguments
 *table*
     The file describing the prisms with record format *x y z_lo z_hi* [ *dx dy* ] [ *rho* ],
     where the optional items are controlled by options **-E** and **-D**, respectively.
-    Density contrasts can be given in kg/m^3 of g/cm^3. **Note**: If **-C** is used then
+    Density contrasts can be given in :math:`\mbox{kg/m}^3` of :math:`\mbox{g/cm}^3`. **Note**: If **-C** is used then
```
Comment (seisman) [inline doc/rst/source/supplements/potential/gravprisms.rst:79]: Should it be 
```
:math:`\mbox{kg/m^3}`
```
?

Comment (PaulWessel) [reply doc/rst/source/supplements/potential/gravprisms.rst:79]: I think mbox is just for plain text in the middle of math, so no?

Comment (seisman) [reply doc/rst/source/supplements/potential/gravprisms.rst:79]: > I think mbox is just for plain text in the middle of math, so no?

You're right. The following expressions:
```
:math:`\mbox{kg/m}^3`

:math:`\mbox{kg/m^3}`

:math:`kg/m^3`
```
are rendered like:
image


```
:math:`\mbox{kg/m}^3`
```
just gives me the wrong impression of (kg/m)^3, not kg/(m^3).

Comment (PaulWessel) [reply doc/rst/source/supplements/potential/gravprisms.rst:79]: OK, so keep as is then.

**Review by seisman — APPROVED** [review]

**Pull request merged**